local_planner/src/nodes/local_planner_node_main.cpp

```diff
@@ -14,27 +14,7 @@ int main(int argc, char** argv) {
   std::thread worker(&LocalPlannerNode::threadFunction, &Node);
 
   std::thread worker_params(&LocalPlannerNode::checkPx4Parameters, &Node);
-  ros::Duration timeout_termination =
```

Comment:
These helper warnings need to be added back

Reply:
The helper warnings here are a artifact from a merge introduced by #274 The helper warnings are still in `local_planner_node.cpp`

Reply:
Rebased just to make it clear